docs/examples/github-actions-cache.md

```diff
@@ -0,0 +1,46 @@
+# Example: GitHub Actions cache
+
+Jobs on Actuated runners start in a clean VM each time. This means dependencies need to be downloaded and build artifacts or caches rebuilt each time. Caching these files in the actions cache can improve workflow execution time.
+
+A lot of the setup actions for package managers have support for caching built-in. See: [setup-node](https://github.com/actions/setup-node#caching-global-packages-data), [setup-python](https://github.com/actions/setup-python#caching-packages-dependencies),... . They require minimal configuration and will create and restore dependency caches for you.
+
+If you have custom workflows that could benefit from caching the cache can be configured manually using the [actions/cache](https://github.com/actions/cache).
+
+Using the actions cache is not limited to GitHub hosted runners but can be used with self-hosted runners. Workflows using the cache action can be converted to run on Actuated runners. You only need to change `runs-on: ubuntu-latest` to `runs-on: actuated`.
+
+## Try out the action on your agent
+
+In this short example we will build [alexellis/registry-creds](https://github.com/alexellis/registry-creds). This is a Kubernetes operator that can be used to replicate Kubernetes ImagePullSecrets to all namespaces.
+
+Create a new file at: `.github/workspaces/build.yaml` and commit it to the repository.
+
+```yaml
+name: build
+
+on: push
+
+jobs:
+  build:
+    runs-on: actuated
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: "alexellis/registry-creds"
+      - name: Setup Golang
+        uses: actions/setup-go@v3
+        with:
+          go-version: ~1.19
+          cache: true
+      - name: Build
+        run: |
+          CGO_ENABLED=0 GO111MODULE=on \
+          go build -ldflags "-s -w -X main.Release=dev -X main.SHA=dev" -o controller
+```
+
+To configure caching with the [setup-go action](https://github.com/actions/setup-go) you only need to set the `cache` input parameter to true.
+
```

Comment:
Add another header to show how to cache without the built-in support. Like we did in the blog post.